build/deployments/docker-compose.yaml

```diff
@@ -42,10 +42,10 @@ services:
       - "9000"
       - "50832"
     ports:
-      - "7081:7081"
-      - "8081:8080"
-      - "9081:9080"
-      - "50832:50832"
+      - "${NODE1_PORT_7081:-0.0.0.0:7081}:7081"
+      - "${NODE1_PORT_8081:-0.0.0.0:8080}:8080"
+      - "${NODE1_PORT_9081:-0.0.0.0:9080}:9080"
```

Comment:
```suggestion
      - "${NODE1_PORT_8081:-0.0.0.0:8081}:8080"
      - "${NODE1_PORT_9081:-0.0.0.0:9081}:9080"
```

Reply:
great suggestion, was also going to point out, I didn't know colloquial names for these ports or else I would have used those. 

But I guess these var names are already long enough and a motivated reader would eventually figure it out.